### ISSUE TITLE
<script>'s src not interpolated

### ISSUE THREAD

**Issue by zaiste** [buxlabs/boxwood#638]
In the following snippet the `src` attribute won't be interpolated:

```html
<script type="text/javascript" src="/{ website.javascript }" />
```

while this one works:

```html
<link rel="stylesheet" href="/{ website.stylesheet }">
```

**Comment by emilos** [comment]
Thanks! It should be simple to add `tags/script.js`

**Comment by zaiste** [comment]
@emilos not sure to understand, could you clarify ? :)

**Comment by emilos** [comment]
Yeah, sorry, just adding some notes. I just meant that the implementation
should be added to that file. Interpolation is done via one of the method
from `utilities/convert.js`, probably `convertAttributes`.

pt., 1 maj 2020, 22:18 użytkownik Jakub Neander <notifications@github.com>
napisał:

**Comment by emilos** [comment]
@zaiste fixed here: https://github.com/buxlabs/boxwood/pull/639/files , released as `v0.32.2`. Thanks again!
